math/vc_aos/include/algebra/math/impl/vc_aos_transform3.hpp

```diff
@@ -36,6 +36,14 @@ using algebra::storage::operator+;
 template <template <typename, std::size_t> class array_t, typename scalar_t>
 struct transform3 {
 
+  // How to address matrix columns
+  enum elem : std::size_t {
+    e_x = 0u,
+    e_y = 1u,
+    e_z = 2u,
+    e_t = 3u,
+  };
```

Comment:
Should we make it private? I'm not too keen on introducing a public "non-class" enumeration. 🤔

Reply:
sure, done